utils/build-script

```diff
@@ -22,6 +22,7 @@ from build_swift import defaults
 from build_swift import driver_arguments
 from build_swift import presets
 from build_swift.migration import migrate_swift_sdks
+import build_swift.postbuild_steps as postbuild_steps
```

Comment:
Why not just use `from build_swift import postbuild_steps` it's the more "Pythonic" way to do this sort of import.